src/main/java/org/truffleruby/interop/InteropNodes.java

```diff
@@ -1095,4 +1095,14 @@ public TruffleObject loggingForeignObject() {
 
     }
 
+    @CoreMethod(names = "single_threaded?", onSingleton = true)
```

Comment:
Should this be in Truffle::Boot, next to similar flags?
single threaded is a property decided by options, not only influencing interop.

Reply:
I'll just remove it since you can get it from the options method.